Buildings/Controls/OBC/CDL/Integers/Sources/TimeTable.mo

```diff
@@ -0,0 +1,207 @@
+    annotation (Line(points={{22,-70},{28,-70}}, color={255,0,255}));
+  connect(not1.y, assMes.u)
+    annotation (Line(points={{52,-70},{58,-70}}, color={255,0,255}));
+  connect(tab.y, realToInteger.u)
+    annotation (Line(points={{-59,0},{-42,0}}, color={0,0,127}));
+  connect(tab.y, add2.u2) annotation (Line(points={{-59,0},{-50,0},{-50,-36},{38,
+          -36}}, color={0,0,127}));
+  annotation (
+defaultComponentName = "intTimTab",
+Documentation(info="<html>
+<p>
+Block that outputs <code>Integer</code> time table values.
+</p>
+<p>
+The block takes as a parameter a time table of a format:
+</p>
+<pre>
+table = [ 0*3600, 2;
+          6*3600, 0;
+          6*3600, 1;
+         18*3600, 8;
+         18*3600, 5;
+         24*3600, -2];
+</pre>
+<p>
+where the first column is time, and the remaining column(s) are the table values.
+The time column contains <code>Real</code> values that are in units of seconds when <code>timeScale</code> equals <code>1</code>.
+<code>timeScale</code> may be used to scale the time values, such that for <code>timeScale = 3600</code> the values 
+in the first column of the table are interpreted as hours.
+<p>
+Any number of columns can be specified.
+</p>
+<p>
+The values in all columns apart from the first column must be of type <code>Integer</code>, otherwise a warning is issued.
+</p>
+<p>
+The parameter <code>smoothness</code> determines how the table values
+are interpolated. The following setting is implemented:
+</p>
+<table summary=\"summary\" border=\"1\" cellspacing=\"0\" cellpadding=\"2\" style=\"border-collapse:collapse;\">
+<tr><th><code>smoothness</code></th><th>Description</th></tr>
+<tr>
+  <td><code>CDL.Types.ConstantSegments</code></td>
+  <td>Table points are not interpolated, but the previous tabulated value is returned.</td>
+</tr>
+</table>
+<p>
+The parameter <code>extrapolation</code> determines how the table
+values are extrapolated. The following settings are allowed:
+</p>
+
+<table summary=\"summary\" border=\"1\" cellspacing=\"0\" cellpadding=\"2\" style=\"border-collapse:collapse;\">
+<tr><th><code>extrapolation</code></th><th>Description</th></tr>
+<tr>
+  <td><code>CDL.Types.HoldLastPoint</code></td>
+  <td>Hold the first or last table point outside of the table scope.</td>
+</tr>
+<tr>
+  <td><code>CDL.Types.Periodic</code></td>
+  <td>Repeat the table scope periodically with periodicity
+      <code>(max(table[:, 1]-min(table[:, 1]))*timeScale)</code>.</td>
+</tr>
+</table>
+
+<p>
+If <code>extrapolation === CDL.Types.Periodic</code>, then the above example
+would give a schedule with periodicity of one day. The simulation can start at any time,
+whether it is a multiple of a day or not, and positive or negative.
+</p>
+
+<p>
+The value of the parameter <code>offset</code> is added to the tabulated
+values.
+The parameters <code>timeScale</code> is used to scale the first column
+of the table. For example, set <code>timeScale = 3600</code> if the first
```

Comment:
Time scale has been explained above already. Make sure you only explain it once.